services/paymaster/src/main.rs

```diff
@@ -1,14 +1,15 @@
 use crate::cli::Cli;
+use crate::config_manager::config::Config;
 use arc_swap::ArcSwap;
 use clap::Parser;
 use opentelemetry::trace::TracerProvider;
 use opentelemetry_otlp::WithExportConfig;
 use std::collections::HashMap;
 use std::sync::Arc;
 use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
-
 mod api;
 mod cli;
+#[allow(dead_code)] // TODO: This module is unused until we bring back the DB
```

Comment:
can you decorate just the load_db_config method and the spawn_config_refersher method?